rust/src/cfd/mod.rs

```diff
@@ -0,0 +1,63 @@
+mod open;
+mod settle;
+
+pub mod models;
+
+pub mod services {
```

Comment:
🙃 Do we need these modules on top? If I am not mistaken all of these calls just route through to an inner call. It might be good enough to just re-export the inner calls as needed in here.

Reply:
no we don't I removed it. 🙂